CHT/HeatFlux.C

```diff
@@ -33,9 +33,13 @@ void preciceAdapter::CHT::HeatFlux::write(double * buffer, bool meshConnectivity
     {
         int patchID = patchIDs_.at(j);
 
-        scalarField gradientPatch=refCast<fixedValueFvPatchScalarField>
-                (T_->boundaryFieldRef()[patchID]
-                 ).snGrad();
+        const scalarField & gradientPatch
```

Comment:
```suggestion
        const scalarField gradientPatch
```
You cannot take the reference here, because the `gradientPatch` is just a temporary created field. We need an explicit copy.

Reply:
Very good point, thank you! Sometimes all we need is a second pair of eyes... :smile: